src/assessment.py

```diff
@@ -50,8 +50,8 @@ def post_assessment():
         )
     except ValueError:
         return "One of the arguments is not parseable as a number", 400
-    except openai.error.InvalidRequestError as e:
```

Comment:
because we are using a plain HTTP library to access openai, there should be no way that we can receive an openai error in response.